app/controllers/return-requirements.controller.js

```diff
@@ -274,6 +283,10 @@ async function submitAgreementsExceptions (request, h) {
     return h.view('return-requirements/agreements-exceptions.njk', pageData)
   }
 
+  // This needs logic
+  flasher(request.yar)
+  flasher(request.yar, 'Added', 'New requirement added')
```

Comment:
Need to work out the logic how this could know it's a new requirement - check session or add a new key/value flag.